opentelemetry-otlp/src/lib.rs

```diff
@@ -119,27 +119,22 @@
 
 use opentelemetry::{global, sdk, trace::TracerProvider};
 
-#[cfg(all(feature = "grpc-sys", not(feature = "tonic")))]
+#[cfg(feature = "grpc-sys")]
 use std::collections::HashMap;
 
 use std::str::FromStr;
 use std::time::Duration;
 
-#[cfg(all(feature = "tonic", not(feature = "integration-testing")))]
+#[cfg(
+not(feature = "integration-testing")
+)]
```

Comment:
This looks a little odd?

Reply:
I think the idea is to hide the proto mod unless we are running integrationt testing.